### ISSUE TITLE
Adding an exercise

### ISSUE THREAD

**Issue by fulldecent** [kabouzeid/Iron#10]
That doesn't exist. This is how it is expecting to add

https://github.com/user-attachments/assets/38f54e93-c6c7-4d47-9069-cf66edea9bf6

That doesn't exist. This is how expecting to add it.

Right now the problem with iron I'd that you have to cancel the work out and delete all the exercises in it and start over if you are doing a new exercise that is not on the list

**Comment by karimknaebel** [comment]
Yeah I agree that would be a good place to add a “create exercise” button.

However, you can already create new exercises without canceling your workout. Just go to the exercises tab -> Custom -> create exercise

**Comment by fulldecent** [comment]
Okay I just figured it out. There is a secret. 

If you start a workout and type in a name then the keyboard opens. And you cannot add new exercises. That is shown in my screenshot. 

You would just need to know that you must hit enter. Then you can proceed. 

This problem persists even if you add an exercise and leave this screen and come back. You are still editing the name. 

----

Recommendation for this separate issue:

Leaving the screen causes keyboard to finish typing. So when you come back you get the full screen. 

And

If you scroll up or down then the typing finishes.

**Comment by lutzzdias** [comment]
Duplicate of #7
